test/e2e/pod_create_test.go

```diff
@@ -876,15 +876,17 @@ ENTRYPOINT ["sleep","99999"]
 
 	It("podman pod create --device", func() {
 		SkipIfRootless("Cannot create devices in /dev in rootless mode")
-		Expect(os.MkdirAll("/dev/foodevdir", os.ModePerm)).To(Succeed())
-		defer os.RemoveAll("/dev/foodevdir")
+		// path must be unique to this test, not used anywhere else
+		devdir := "/dev/devdirpodcreate"
```

Comment:
There is nothing requiring these to be in /dev either.  You could just put them on the PODMAN_TMPDIR.

Reply:
True for two of these three tests. Not obviously true for the `run_privileged` one, and I chose simplicity of review over a full revamp.

This is probably an illustration of today's scrum conversation, about doing the right thing vs the expedient one. I tentatively side with Voltaire on this one, but will spend the effort tomorrow to do a proper cleanup if you'd like me to.

Reply:
Well the run_privileged one would work as well as long as you looked for the device in `devdir` instead of /dev.